include/onnxruntime/core/framework/op_kernel_context.h

```diff
@@ -253,4 +255,42 @@ inline SparseTensor* OpKernelContext::Output<SparseTensor>(int index) {
 }
 #endif
 
+// For invoking kernels without a graph
+class InstantKernelContext : public OpKernelContext {
```

Comment:
Call it GraphlessKernelContext? 
Also, can you put this in a file of its own?

Reply:
Moved to standalone_op_invocker.cc